
2019-05-28 21:04:32
[ERROR]-[Thread: main]-[com.alibaba.druid.pool.DruidDataSource.init()]: init datasource error, url: jdbc:mysql://192.168.15.141/37duc?characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&serverTimezone=UTC&serverTimezone=Asia/Shanghai
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:590)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:57)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:861)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1229)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1225)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:90)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:43)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:35)
Caused by: com.mysql.cj.core.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:133)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:149)
	at com.mysql.cj.mysqla.io.MysqlaSocketConnection.connect(MysqlaSocketConnection.java:83)
	at com.mysql.cj.mysqla.MysqlaSession.connect(MysqlaSession.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:1726)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1596)
	... 33 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at com.mysql.cj.core.io.StandardSocketFactory.connect(StandardSocketFactory.java:202)
	at com.mysql.cj.mysqla.io.MysqlaSocketConnection.connect(MysqlaSocketConnection.java:57)
	... 36 more

2019-05-28 21:04:32
[ERROR]-[Thread: main]-[com.alibaba.druid.pool.DruidDataSource.init()]: {dataSource-1} init error
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:590)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:57)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:861)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1229)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1225)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:90)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:43)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:35)
Caused by: com.mysql.cj.core.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:133)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:149)
	at com.mysql.cj.mysqla.io.MysqlaSocketConnection.connect(MysqlaSocketConnection.java:83)
	at com.mysql.cj.mysqla.MysqlaSession.connect(MysqlaSession.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:1726)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1596)
	... 33 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at com.mysql.cj.core.io.StandardSocketFactory.connect(StandardSocketFactory.java:202)
	at com.mysql.cj.mysqla.io.MysqlaSocketConnection.connect(MysqlaSocketConnection.java:57)
	... 36 more

2019-05-28 21:04:32
[ERROR]-[Thread: main]-[com.jfinal.core.Config.startPlugins()]: Plugin start error: com.jfinal.plugin.activerecord.ActiveRecordPlugin. 
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
com.jfinal.plugin.activerecord.ActiveRecordException: com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:55)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:35)
Caused by: com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:590)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:57)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1606)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:633)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:861)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1229)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1225)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:90)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:43)
	... 18 more
Caused by: com.mysql.cj.core.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:133)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:149)
	at com.mysql.cj.mysqla.io.MysqlaSocketConnection.connect(MysqlaSocketConnection.java:83)
	at com.mysql.cj.mysqla.MysqlaSession.connect(MysqlaSession.java:122)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:1726)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:1596)
	... 33 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at com.mysql.cj.core.io.StandardSocketFactory.connect(StandardSocketFactory.java:202)
	at com.mysql.cj.mysqla.io.MysqlaSocketConnection.connect(MysqlaSocketConnection.java:57)
	... 36 more

2019-05-28 21:15:40
[WARN]-[Thread: XNIO-1 task-1]-[com.jfinal.core.ActionHandler.handle()]: 404 Action Not Found: /admin

2019-05-28 21:32:04
[WARN]-[Thread: XNIO-2 task-1]-[com.jfinal.core.ActionHandler.handle()]: 404 Action Not Found: /admin

2019-05-28 21:32:05
[WARN]-[Thread: XNIO-2 task-2]-[com.jfinal.core.ActionHandler.handle()]: 404 Action Not Found: /admin

2019-05-28 21:33:38
[WARN]-[Thread: XNIO-1 task-1]-[com.jfinal.core.ActionHandler.handle()]: 404 Action Not Found: /admin

2019-05-28 21:39:54
[ERROR]-[Thread: XNIO-1 task-8]-[com.jfinal.core.ActionHandler.handle()]: /admin/login/doLogin
java.lang.RuntimeException: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.jfinal.aop.Invocation.invoke(Invocation.java:96)
	at com.jfinal.aop.Callback.intercept(Callback.java:96)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.login(<generated>)
	at com.crm.controller.admin.AdminLoginController.doLogin(AdminLoginController.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:75)
	at com.jfinal.validate.Validator.intercept(Validator.java:86)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:69)
	at com.jfinal.core.ActionHandler.handle(ActionHandler.java:81)
	at com.jfinal.core.JFinalFilter.doFilter(JFinalFilter.java:86)
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:360)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.CGLIB$login$4(<generated>)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430$$FastClassByCGLIB$$26c38efe.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:84)
	... 42 more

2019-05-28 21:39:59
[ERROR]-[Thread: XNIO-1 task-11]-[com.jfinal.core.ActionHandler.handle()]: /admin/login/doLogin
java.lang.RuntimeException: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.jfinal.aop.Invocation.invoke(Invocation.java:96)
	at com.jfinal.aop.Callback.intercept(Callback.java:96)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.login(<generated>)
	at com.crm.controller.admin.AdminLoginController.doLogin(AdminLoginController.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:75)
	at com.jfinal.validate.Validator.intercept(Validator.java:86)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:69)
	at com.jfinal.core.ActionHandler.handle(ActionHandler.java:81)
	at com.jfinal.core.JFinalFilter.doFilter(JFinalFilter.java:86)
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:360)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.CGLIB$login$4(<generated>)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430$$FastClassByCGLIB$$26c38efe.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:84)
	... 42 more

2019-05-28 21:40:30
[ERROR]-[Thread: XNIO-1 task-14]-[com.jfinal.core.ActionHandler.handle()]: /admin/login/doLogin
java.lang.RuntimeException: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.jfinal.aop.Invocation.invoke(Invocation.java:96)
	at com.jfinal.aop.Callback.intercept(Callback.java:96)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.login(<generated>)
	at com.crm.controller.admin.AdminLoginController.doLogin(AdminLoginController.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:75)
	at com.jfinal.validate.Validator.intercept(Validator.java:86)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:69)
	at com.jfinal.core.ActionHandler.handle(ActionHandler.java:81)
	at com.jfinal.core.JFinalFilter.doFilter(JFinalFilter.java:86)
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:360)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.CGLIB$login$4(<generated>)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430$$FastClassByCGLIB$$26c38efe.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:84)
	... 42 more

2019-05-28 21:44:54
[ERROR]-[Thread: XNIO-1 task-5]-[com.jfinal.core.ActionHandler.handle()]: /admin/login/doLogin
java.lang.RuntimeException: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.jfinal.aop.Invocation.invoke(Invocation.java:96)
	at com.jfinal.aop.Callback.intercept(Callback.java:96)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.login(<generated>)
	at com.crm.controller.admin.AdminLoginController.doLogin(AdminLoginController.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:75)
	at com.jfinal.validate.Validator.intercept(Validator.java:86)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:69)
	at com.jfinal.core.ActionHandler.handle(ActionHandler.java:81)
	at com.jfinal.core.JFinalFilter.doFilter(JFinalFilter.java:86)
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:360)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: java.lang.Object.login(Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;)Lcom/jfinal/kit/Ret;
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430.CGLIB$login$4(<generated>)
	at com.crm.service.system.AdminLoginService$$EnhancerByCGLIB$$c0323430$$FastClassByCGLIB$$26c38efe.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:84)
	... 42 more

2019-05-28 21:47:15
[ERROR]-[Thread: XNIO-1 task-4]-[com.jfinal.core.ActionHandler.handle()]: /admin/login/doLogin
java.lang.NullPointerException
	at com.jfinal.plugin.activerecord.Model.find(Model.java:664)
	at com.jfinal.plugin.activerecord.Model.findFirst(Model.java:682)
	at com.crm.service.system.impl.AdminLoginServiceImpl.login(AdminLoginServiceImpl.java:28)
	at com.crm.service.system.impl.AdminLoginServiceImpl$$EnhancerByCGLIB$$bb84627a.CGLIB$login$0(<generated>)
	at com.crm.service.system.impl.AdminLoginServiceImpl$$EnhancerByCGLIB$$bb84627a$$FastClassByCGLIB$$e5076052.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:84)
	at com.jfinal.aop.Callback.intercept(Callback.java:96)
	at com.crm.service.system.impl.AdminLoginServiceImpl$$EnhancerByCGLIB$$bb84627a.login(<generated>)
	at com.crm.controller.admin.AdminLoginController.doLogin(AdminLoginController.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:75)
	at com.jfinal.validate.Validator.intercept(Validator.java:86)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:69)
	at com.jfinal.core.ActionHandler.handle(ActionHandler.java:81)
	at com.jfinal.core.JFinalFilter.doFilter(JFinalFilter.java:86)
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:360)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

2019-05-28 21:51:59
[ERROR]-[Thread: main]-[com.jfinal.core.Config.startPlugins()]: Plugin start error: com.jfinal.plugin.activerecord.ActiveRecordPlugin. 
java.sql.SQLSyntaxErrorException: Table 'crm.sys_config' doesn't exist
com.jfinal.plugin.activerecord.ActiveRecordException: java.sql.SQLSyntaxErrorException: Table 'crm.sys_config' doesn't exist
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:55)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:33)
Caused by: java.sql.SQLSyntaxErrorException: Table 'crm.sys_config' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:536)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115)
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1983)
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1936)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1422)
	at com.alibaba.druid.filter.FilterChainImpl.statement_executeQuery(FilterChainImpl.java:2831)
	at com.alibaba.druid.filter.FilterAdapter.statement_executeQuery(FilterAdapter.java:2503)
	at com.alibaba.druid.filter.FilterEventAdapter.statement_executeQuery(FilterEventAdapter.java:302)
	at com.alibaba.druid.filter.FilterChainImpl.statement_executeQuery(FilterChainImpl.java:2828)
	at com.alibaba.druid.wall.WallFilter.statement_executeQuery(WallFilter.java:527)
	at com.alibaba.druid.filter.FilterChainImpl.statement_executeQuery(FilterChainImpl.java:2828)
	at com.alibaba.druid.proxy.jdbc.StatementProxyImpl.executeQuery(StatementProxyImpl.java:221)
	at com.alibaba.druid.pool.DruidPooledStatement.executeQuery(DruidPooledStatement.java:291)
	at com.jfinal.plugin.activerecord.TableBuilder.doBuild(TableBuilder.java:71)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:47)
	... 18 more

2019-05-28 21:53:52
[WARN]-[Thread: XNIO-1 task-6]-[com.jfinal.plugin.ehcache.CacheKit.getOrAddCache()]: Could not find cache config [loginAdmin], using default.

2019-05-28 21:53:53
[ERROR]-[Thread: XNIO-1 task-7]-[com.jfinal.core.ActionHandler.handle()]: /admin/index
java.lang.RuntimeException: java.lang.NoSuchMethodError: java.lang.Object.findByUserId(Ljava/lang/Long;I)Ljava/util/List;
	at com.jfinal.aop.Invocation.invoke(Invocation.java:96)
	at com.jfinal.aop.Callback.intercept(Callback.java:96)
	at com.crm.service.system.PermissionService$$EnhancerByCGLIB$$14534029.findByUserId(<generated>)
	at com.crm.interceptor.AdminLoginSessionInterceptor.intercept(AdminLoginSessionInterceptor.java:41)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:69)
	at com.jfinal.core.ActionHandler.handle(ActionHandler.java:81)
	at com.jfinal.core.JFinalFilter.doFilter(JFinalFilter.java:86)
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138)
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272)
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81)
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:360)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NoSuchMethodError: java.lang.Object.findByUserId(Ljava/lang/Long;I)Ljava/util/List;
	at com.crm.service.system.PermissionService$$EnhancerByCGLIB$$14534029.CGLIB$findByUserId$10(<generated>)
	at com.crm.service.system.PermissionService$$EnhancerByCGLIB$$14534029$$FastClassByCGLIB$$ae1599ad.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228)
	at com.jfinal.aop.Invocation.invoke(Invocation.java:84)
	... 36 more

2019-05-28 21:55:48
[ERROR]-[Thread: main]-[com.alibaba.druid.pool.DruidDataSource.init()]: init datasource error, url: jdbc:mysql://127.0.0.1/crm?characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&serverTimezone=UTC&serverTimezone=Asia/Shanghai&useSSL=trues
java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:505)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:479)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:489)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:78)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:629)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:861)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1229)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1225)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:90)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:43)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:33)
Caused by: com.mysql.cj.core.exceptions.WrongArgumentException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:154)
	at com.mysql.cj.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:2310)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:620)
	... 32 more
Caused by: com.mysql.cj.core.exceptions.CJException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:73)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
	at com.mysql.cj.core.conf.AbstractPropertyDefinition.validateAllowableValues(AbstractPropertyDefinition.java:186)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:45)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:29)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.setFromString(AbstractRuntimeProperty.java:91)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:86)
	at com.mysql.cj.core.conf.ModifiableBooleanProperty.initializeFrom(ModifiableBooleanProperty.java:42)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:70)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:151)
	... 34 more

2019-05-28 21:55:48
[ERROR]-[Thread: main]-[com.alibaba.druid.pool.DruidDataSource.init()]: {dataSource-1} init error
java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:505)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:479)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:489)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:78)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:629)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:861)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1229)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1225)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:90)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:43)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:33)
Caused by: com.mysql.cj.core.exceptions.WrongArgumentException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:154)
	at com.mysql.cj.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:2310)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:620)
	... 32 more
Caused by: com.mysql.cj.core.exceptions.CJException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:73)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
	at com.mysql.cj.core.conf.AbstractPropertyDefinition.validateAllowableValues(AbstractPropertyDefinition.java:186)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:45)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:29)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.setFromString(AbstractRuntimeProperty.java:91)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:86)
	at com.mysql.cj.core.conf.ModifiableBooleanProperty.initializeFrom(ModifiableBooleanProperty.java:42)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:70)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:151)
	... 34 more

2019-05-28 21:55:48
[ERROR]-[Thread: main]-[com.jfinal.core.Config.startPlugins()]: Plugin start error: com.jfinal.plugin.activerecord.ActiveRecordPlugin. 
java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
com.jfinal.plugin.activerecord.ActiveRecordException: java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:55)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:33)
Caused by: java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:505)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:479)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:489)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:78)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:629)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:861)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1229)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:1225)
	at com.alibaba.druid.pool.DruidDataSource.getConnection(DruidDataSource.java:90)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:43)
	... 18 more
Caused by: com.mysql.cj.core.exceptions.WrongArgumentException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:154)
	at com.mysql.cj.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:2310)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:620)
	... 32 more
Caused by: com.mysql.cj.core.exceptions.CJException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:73)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
	at com.mysql.cj.core.conf.AbstractPropertyDefinition.validateAllowableValues(AbstractPropertyDefinition.java:186)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:45)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:29)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.setFromString(AbstractRuntimeProperty.java:91)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:86)
	at com.mysql.cj.core.conf.ModifiableBooleanProperty.initializeFrom(ModifiableBooleanProperty.java:42)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:70)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:151)
	... 34 more

2019-05-28 21:55:48
[ERROR]-[Thread: Druid-ConnectionPool-Create-1496220730]-[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run()]: create connection SQLException, url: jdbc:mysql://127.0.0.1/crm?characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&serverTimezone=UTC&serverTimezone=Asia/Shanghai&useSSL=trues, errorCode 0, state S1009
java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:505)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:479)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:489)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:78)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:629)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2468)
Caused by: com.mysql.cj.core.exceptions.WrongArgumentException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:154)
	at com.mysql.cj.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:2310)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:620)
	... 10 more
Caused by: com.mysql.cj.core.exceptions.CJException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:73)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
	at com.mysql.cj.core.conf.AbstractPropertyDefinition.validateAllowableValues(AbstractPropertyDefinition.java:186)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:45)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:29)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.setFromString(AbstractRuntimeProperty.java:91)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:86)
	at com.mysql.cj.core.conf.ModifiableBooleanProperty.initializeFrom(ModifiableBooleanProperty.java:42)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:70)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:151)
	... 12 more

2019-05-28 21:55:48
[ERROR]-[Thread: Druid-ConnectionPool-Create-1496220730]-[com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run()]: create connection SQLException, url: jdbc:mysql://127.0.0.1/crm?characterEncoding=utf8&zeroDateTimeBehavior=convertToNull&serverTimezone=UTC&serverTimezone=Asia/Shanghai&useSSL=trues, errorCode 0, state S1009
java.sql.SQLException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:545)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:505)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:479)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:489)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:78)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:629)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:347)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:219)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:156)
	at com.alibaba.druid.filter.stat.StatFilter.connection_connect(StatFilter.java:218)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.filter.FilterAdapter.connection_connect(FilterAdapter.java:786)
	at com.alibaba.druid.filter.FilterChainImpl.connection_connect(FilterChainImpl.java:150)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1560)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1623)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2468)
Caused by: com.mysql.cj.core.exceptions.WrongArgumentException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:93)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:154)
	at com.mysql.cj.jdbc.ConnectionImpl.initializeDriverProperties(ConnectionImpl.java:2310)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:620)
	... 10 more
Caused by: com.mysql.cj.core.exceptions.CJException: The connection property 'useSSL' only accepts values of the form: 'true', 'false', 'yes' or 'no'. The value 'trues' is not in this set.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:54)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:73)
	at com.mysql.cj.core.exceptions.ExceptionFactory.createException(ExceptionFactory.java:62)
	at com.mysql.cj.core.conf.AbstractPropertyDefinition.validateAllowableValues(AbstractPropertyDefinition.java:186)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:45)
	at com.mysql.cj.core.conf.BooleanPropertyDefinition.parseObject(BooleanPropertyDefinition.java:29)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.setFromString(AbstractRuntimeProperty.java:91)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:86)
	at com.mysql.cj.core.conf.ModifiableBooleanProperty.initializeFrom(ModifiableBooleanProperty.java:42)
	at com.mysql.cj.core.conf.AbstractRuntimeProperty.initializeFrom(AbstractRuntimeProperty.java:70)
	at com.mysql.cj.core.conf.DefaultPropertySet.initializeProperties(DefaultPropertySet.java:151)
	... 12 more

2019-05-28 21:56:20
[WARN]-[Thread: XNIO-1 task-1]-[com.jfinal.plugin.ehcache.CacheKit.getOrAddCache()]: Could not find cache config [loginAdmin], using default.

2019-05-28 22:06:00
[WARN]-[Thread: XNIO-1 task-19]-[com.jfinal.core.ActionHandler.handle()]: 404 Action Not Found: /admin/system/user

2019-05-28 22:06:07
[WARN]-[Thread: XNIO-1 task-18]-[com.jfinal.core.ActionHandler.handle()]: 404 Action Not Found: /admin/system/role

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:19
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:11:32
[WARN]-[Thread: XNIO-1 task-1]-[com.jfinal.plugin.ehcache.CacheKit.getOrAddCache()]: Could not find cache config [loginAdmin], using default.

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:12:52
[WARN]-[Thread: HotSwapWatcher]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:14:03
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:19
[ERROR]-[Thread: main]-[com.jfinal.core.Config.startPlugins()]: Plugin start error: com.jfinal.plugin.activerecord.ActiveRecordPlugin. 
java.sql.SQLSyntaxErrorException: Table 'crm.sys_log_operate' doesn't exist
com.jfinal.plugin.activerecord.ActiveRecordException: java.sql.SQLSyntaxErrorException: Table 'crm.sys_log_operate' doesn't exist
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:55)
	at com.jfinal.plugin.activerecord.ActiveRecordPlugin.start(ActiveRecordPlugin.java:226)
	at com.jfinal.core.Config.startPlugins(Config.java:128)
	at com.jfinal.core.Config.configPluginWithOrder(Config.java:71)
	at com.jfinal.core.Config.configJFinal(Config.java:56)
	at com.jfinal.core.JFinal.init(JFinal.java:61)
	at com.jfinal.core.JFinalFilter.init(JFinalFilter.java:62)
	at io.undertow.servlet.core.LifecyleInterceptorInvocation.proceed(LifecyleInterceptorInvocation.java:111)
	at io.undertow.servlet.core.ManagedFilter.createFilter(ManagedFilter.java:80)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:589)
	at io.undertow.servlet.core.DeploymentManagerImpl$2.call(DeploymentManagerImpl.java:554)
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:42)
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at io.undertow.servlet.core.DeploymentManagerImpl.start(DeploymentManagerImpl.java:596)
	at com.jfinal.server.undertow.UndertowServer.configHttp(UndertowServer.java:270)
	at com.jfinal.server.undertow.UndertowServer.doStart(UndertowServer.java:248)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:146)
	at com.jfinal.server.undertow.UndertowServer.start(UndertowServer.java:80)
	at com.common.CoreConfig.main(CoreConfig.java:33)
Caused by: java.sql.SQLSyntaxErrorException: Table 'crm.sys_log_operate' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:536)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:513)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:115)
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1983)
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:1936)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1422)
	at com.alibaba.druid.filter.FilterChainImpl.statement_executeQuery(FilterChainImpl.java:2831)
	at com.alibaba.druid.filter.FilterAdapter.statement_executeQuery(FilterAdapter.java:2503)
	at com.alibaba.druid.filter.FilterEventAdapter.statement_executeQuery(FilterEventAdapter.java:302)
	at com.alibaba.druid.filter.FilterChainImpl.statement_executeQuery(FilterChainImpl.java:2828)
	at com.alibaba.druid.wall.WallFilter.statement_executeQuery(WallFilter.java:527)
	at com.alibaba.druid.filter.FilterChainImpl.statement_executeQuery(FilterChainImpl.java:2828)
	at com.alibaba.druid.proxy.jdbc.StatementProxyImpl.executeQuery(StatementProxyImpl.java:221)
	at com.alibaba.druid.pool.DruidPooledStatement.executeQuery(DruidPooledStatement.java:291)
	at com.jfinal.plugin.activerecord.TableBuilder.doBuild(TableBuilder.java:71)
	at com.jfinal.plugin.activerecord.TableBuilder.build(TableBuilder.java:47)
	... 18 more

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:40
[WARN]-[Thread: main]-[com.jfinal.core.paragetter.ParaProcessorBuilder.createParaGetter()]: you should add compiler flag -parameters to support parameter auto binding

2019-05-28 22:15:52
[WARN]-[Thread: XNIO-1 task-1]-[com.jfinal.plugin.ehcache.CacheKit.getOrAddCache()]: Could not find cache config [loginAdmin], using default.
